dvc/repo/experiments/executor/base.py

```diff
@@ -248,6 +250,28 @@ def on_diverged_ref(orig_ref: str, new_rev: str):
             )
         return refs
 
+    @classmethod
+    def _auto_push_check(cls, dvc: "Repo"):
```

Comment:
This is minor, but I would maybe call this function `_validate_remotes` or something along those lines. The way the PR looks right now it comes across as if this is going to be a modified version or wrapper around `_auto_push()`, when it's actually a function to validate whether or not we have git and dvc remotes to push to